client/checkout/woopay/style.scss

```diff
@@ -270,3 +270,50 @@
 		}
 	}
 }
+
+@keyframes spinner__animation {
```

Comment:
Should we place this in `client/stylesheets/abstracts/_mixins.scss`?

Reply:
We discussed this and figured it could be left here since it won't be used elsewhere. 👍